nnpdf_data/nnpdf_data/commondata/DYE605_Z0_38P8GEV_DW/metadata.yaml

```diff
@@ -59,5 +59,8 @@ implemented_observables:
     legacy:
       data_uncertainties:
       - uncertainties_legacy_PXSEC.yaml
-  data_central: data_legacy_PXSEC.yaml
+    reimplemented:
+      data_uncertainties:
+      - uncertainties_reimplemented_PXSEC.yaml
+  data_central: data_reimplemented_PXSEC.yaml
```

Comment:
Some final comments. This should not be `reimplemented`, this should just be the normal `data_uncertainties`, not a variant.

Same for all the others.

Reply:
we want to keep the old data under `legacy` for that dataset, and the new implementation be the default for all (so not keep `reimplemented` for any)

Reply:
Sorry for the delay in this, but now I think everything is cleaned up.

Reply:
Thanks. You kept the `reimplemented` variant though (although you removed it from the names). Maybe you didn't commit that part?